api/prisma/seed.ts

```diff
@@ -1,6 +1,6 @@
 import htmlToText from 'html-to-text';
 import s3 from '../src/lib/s3';
-
+import { createQueue, getQueue } from '../src/lib/sqs';
```

Comment:
import * as sqs from '../src/lib/sqs';

or

import sqs from '../src/lib/sqs';

Please check coding conventions on the readme file https://github.com/JiscSD/octopus#team-code-conventions